packages/api/src/schemas/user/resolver.ts

```diff
@@ -0,0 +1,68 @@
+import { withFilter } from 'aws-lambda-graphql';
+import {
+  MutationCreateUserArgs,
+  MutationDeleteUserArgs,
+  MutationUpdateUserArgs,
+  PaginatedUsers,
+  QueryGetUserArgs,
+  QueryListUsersArgs,
+  SubscribeToUser,
+  SubscriptionSubscribeToUserArgs,
+  User,
+} from '@sf/core/user';
+import { pubSub } from '@sf/api/config';
+import { userService } from '@sf/api/services';
+import { PUBLISH_USER, PublishUserPayload } from '@sf/api/subscriptions';
+
+const resolver = {
+  Query: {
+    async listUsers(_, args: QueryListUsersArgs): Promise<PaginatedUsers> {
+      const { filter, pageInfo } = args;
+      const { items, offset, total } = await userService.findAll(pageInfo, filter);
+      return {
+        offset,
+        total,
+        users: items,
+      };
+    },
```

Comment:
How do you see this file on a large GraphQL API?

Reply:
We can split resolvers for `Query`, `Mutation` and `Subscription` into smaller files and test them.
That's the pattern I have been using and it worked well for me as a readable code and easily-testable code.